run_kind_ci.sh

```diff
@@ -154,6 +154,7 @@ set_default_params() {
   export WORKSPACE=${WORKSPACE:-"/tmp/kind_ci/$PROJECT"}
   export KIND_NUM_WORKER=${KIND_NUM_WORKER:-2}
   export KUBECONFIG=${KUBECONFIG:-$HOME/admin.conf}
+  export NODE_IMAGE="$NODE_IMAGE"
```

Comment:
`KIND_IMAGE` would be more suitable

Reply:
Will change it to `KIND_NODE_IMAGE`, i think this is the most appropriate. WDYT?

Reply:
changed it to KIND_NODE_IMAGE